core/blockchain/impl/block_tree_impl.hpp

```diff
@@ -55,6 +56,9 @@ namespace kagome::blockchain {
        */
       std::shared_ptr<TreeNode> getByHash(const primitives::BlockHash &hash);
 
+      boost::optional<std::vector<std::shared_ptr<TreeNode>>> getWayTo(
```

Comment:
Might getPathTo be a bit more conventional? AFAIK in graphs and in particular trees this is usually called a path, if I am not mistaken about what this method does.

Reply:
Yep! Thanks...i forgot this word:)